### PR TITLE
Feature/linters

### PR DESCRIPTION
**Pull request by Markotweyo**
Portfolio project in html/css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            </div>
+            <nav class="icon-menu">
+                    <div class="menu-icon">
+                        <img src="./images/menu.svg" alt="menu icon" class="nav-icon">
+                    </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1 class="username">I'm Mark, Glad to see you!</h1>
```
Comment (zainsadaqat) [inline index.html:27]: Please use the font family "Poppins" as per requirement for the heading. 👍🏻 
[Your Style]
Screen Shot 2022-04-25 at 4 03 11 PM

[Required Syle]
Screen Shot 2022-04-25 at 4 01 47 PM

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            </div>
+            <div class="description">
+                <p class="user-description">
+                    I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (zainsadaqat) [inline index.html:31]: Please use the font family "Poppins" as per requirement for the paragraph. 👍🏻 

[Font, you used]
Screen Shot 2022-04-25 at 4 03 40 PM

[Required Font]
Screen Shot 2022-04-25 at 4 04 33 PM

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                </p>
+            </div>
+            <span class="socialmedia">LET'S CONNECT</span>
```
Comment (zainsadaqat) [inline index.html:34]: Please use the font family "Poppins" as per requirement for the subtitle. 👍🏻 

[Font Used]
Screen Shot 2022-04-25 at 4 05 49 PM


[Font Required]
Screen Shot 2022-04-25 at 4 06 13 PM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @Markotweyo  😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] Please use full background as per given figma design

[Required Design]

Screen Shot 2022-04-25 at 4 08 32 PM

[Your Background]

Screen Shot 2022-04-25 at 4 08 53 PM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Markotweyo** [comment]
Hi, I have fixed the background cover image and the "Poppins" font family

File: index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                <a href="" class="linkedin-link">
+                    <img src="./images/linkedin.svg" alt="linkedin" class="linkedin" width="20px" height="20px">
+                </a>
+                <a href="" class="email-link">
+                    <img src="./images/email.svg" alt="email" class="email" width="20px" height="20px">
+                </a>
+                <a href="" class="social-media-link">
+                    <img src="./images/github.svg" alt="github" class="github" width="20px" height="20px" >
+                </a> 
+            </div>
```
Comment (Lucas-Masaba) [inline index.html:48]: [OPTIONAL]
- Please remember to place these icons in an `ul` tag with each icon having a `li` tag.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Markotweyo  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Good job implementing the changes by the previous reviewer 🏄 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**